ViewModel/BrowserViewModel.swift

```diff
@@ -179,7 +180,7 @@ class BrowserViewModel: NSObject, ObservableObject,
         } else if url.isKiwixURL {
             decisionHandler(.allow)
         } else if url.scheme == "http" || url.scheme == "https" {
-            NotificationCenter.default.post(name: .externalLink, object: nil, userInfo: ["url": url])
```

Comment:
Could you remove this as well? (I don't think we need this anymore, right?)

https://github.com/kiwix/apple/blob/fa397a905f95b946e1cf6b55328771b101bb91f8/SwiftUI/Patches.swift#L65